### DISCUSSION TITLE
Zimodem (Retro Rewind Edition)

### DISCUSSION THREAD

**Discussion by alexrepp** [bozimmerman/Zimodem#119] [Q&A]
![zimodem retrorewind](https://github.com/bozimmerman/Zimodem/assets/156201608/d2e23511-bacd-445a-9e02-091fa84a807e)
Hello Community - sry am nearly a noob in programming, but can anybody tell me, what's wrong in my Zimodem Settings ?
Am trying to use RetroTerm with turbo56k to check out some c64 56k BBSs - usually got the response, that i need a turbo56k terminal after connecting to the server and it diconnects again - but i'm definitvely using such a terminal ... then i found out, that even PING gives me an ERROR ... so maybe a "Nerd" gets my Problem on the first sight, if he/she/it sees my settings at my hopefully attached file?
Regards, Alex

**Comment by retrocomputacion** [comment]
Your configuration looks ok at first glance, just make sure to use ATD instead of ATDT when connecting  to a Turbo56K BBS.

**Comment by alexrepp** [comment]
hi and ty for your quick response :) guess i get that "wrong terminal"-msg only if i try atdt - but that man at youtube (^^) does so ... guess i#ve tried atd before too, and after a cruel 5 minutes loading process i've already been in your (?) BBS ... but that worked only 1 of 20 attempts - the rest got stuck .... tried it out again yesterday again and at 2 of 10 attempts i came to the turbo56k picture - but not further on. My picture is a small part of that cruel login process - i have to hit enter felt 300 times by creating mysterious symbols till it goes on alone ... but really guess, theres some setting wrong in my modem, if i can't even PING google 
![IMG_20240111_001410_437](https://github.com/bozimmerman/Zimodem/assets/156201608/b2bc4582-8668-4229-b6cf-575abda406fb)
![IMG_20240111_001528_880](https://github.com/bozimmerman/Zimodem/assets/156201608/d29848f1-2001-4007-bc1d-b56bb51644d8)

**Reply by retrocomputacion** [reply]
I'm the developer of Retroterm and RetroBBS, I dont run any BBS myself.
The error in the screenshot is a known issue in an old version of the BBS, is not a problem on your side. (The problem is related with latency so it is hard to reproduce and wasn't discovered before released back then) 
To which BBS are you trying to connect? Of the ones I know of only one is running right now (kilokahn.myds.me:6404) and is using a much newer version of the software.

**Reply by alexrepp** [reply]
ok ty - it been the lu8fjh-c64.ddns.net:6402 ... but one of all attempts it really worked till it got stuck at a SID stream :/ - thanks for the link, I#ll give it a try :)

**Reply by retrocomputacion** [reply]
Ah yes, lu8fjh is using a very old version.
LU4FBU.DDNS.NET:6400 and lab64.ddns.net:6400 are online too. Both are using almost up to date BBS software

**Comment by alexrepp** [comment]
i'll try resetting my firmware - maybe i did some mistakes by mistake ^^ ... can anybody tell me what "Use &F86 to reformat the SPIFFS" causes and how to use it: "AT&F86"?
thanks in advance, alex
